src/sentry/lang/native/symbolicator.py

```diff
@@ -352,6 +352,12 @@ def get_sources_for_project(project):
 
     sources = []
 
+    def remove_ignored_sources(sources):
```

Comment:
This is all nitpicking, so do with it as you like.
* I think naming-wise this is what is usually known as a filter, so I'd go with `filter_ignored_sources()`.
* This doesn't use any free variables so don't need to be a closure.  I'd make it a top-level function instead of nesting it.  This means you can unittest it easily etc.

Reply:
Sorry for dragging this out; While pulling `remove_ignored_sources` out I realized I'd completely missed something in my implementation: It doesn't acknowledge aliases at all. I figure ops would probably (hopefully) be more likely to use aliases with this kill switch, so I changed this implementation a little.

The new logic recognizes and filters for both aliases and de-aliased internal source IDs and pulls all of the filtering logic out into its own top-level function, making it pretty unit testable. This still gets covered in the integration tests seeing as I'd accidentally broken them in some of my commits, so I think we're good there.

Seeing as I'd reworked the logic quite a bit, could I bother you to take _yet another look_ at this? I'm mostly hoping that you could check to make sure I haven't made any errors handling aliases, and if there's something else I might've missed with source IDs like aliases.